atc/db/worker_resource_cache.go

```diff
@@ -32,32 +35,40 @@ func (workerResourceCache WorkerResourceCache) FindOrCreate(tx Tx) (*UsedWorkerR
 		return nil, ErrWorkerBaseResourceTypeDisappeared
 	}
 
-	id, found, err := workerResourceCache.find(tx, usedWorkerBaseResourceType)
+	ids, workerBaseResourceTypeIds, found, err := workerResourceCache.find(tx, workerResourceCache.WorkerName)
```

Comment:
Can we get rid of the `workerName` param here?

Reply:
Yes